modules/apps/commerce/commerce-currency-web/src/main/java/com/liferay/commerce/currency/web/internal/display/context/CommerceCurrenciesDisplayContext.java

```diff
@@ -236,25 +236,45 @@ else if (navigation.equals("inactive")) {
 				getOrderByCol(), getOrderByType()));
 		_searchContainer.setOrderByType(getOrderByType());
 
-		if (active != null) {
-			boolean navigationActive = active;
-
-			_searchContainer.setResultsAndTotal(
-				() -> _commerceCurrencyService.getCommerceCurrencies(
-					themeDisplay.getCompanyId(), navigationActive,
-					_searchContainer.getStart(), _searchContainer.getEnd(),
-					_searchContainer.getOrderByComparator()),
-				_commerceCurrencyService.getCommerceCurrenciesCount(
-					themeDisplay.getCompanyId(), navigationActive));
+		if (!StringUtil.equals(getKeywords(), "")) {
+			if (active != null) {
+				boolean navigationActive = active;
+
+				_searchContainer.setResultsAndTotal(
```

Comment:
This is another one I guess where we should replace the search container.
@andrea-ale-sbarra